guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<p style="user-select: auto;">The first two bullets appear to have hyperlink issues. It could just be ReviewNB formatting issue</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Added the links and formatting looks correct in notebook.</p>